### ISSUE TITLE
Ember Inspector `4.6.1` fails to load in latest Firefox (Release & Developer Editions)

### ISSUE THREAD

**Issue by andrewobrien** [emberjs/ember-inspector#2104]
**Describe the bug**
Ember inspector latest release `4.6.1` fails to load in Firefox `103.0.2` & Firefox Developer Edition `104.0b10` (latest releases). 

**To Reproduce**
1. Visit https://music.apple.com/us/browse
2. Try to open Ember Inspector tab in developer tools

**Expected behavior**
Ember Inspector will load

**Screenshots**
<img width="1256" alt="CleanShot 2022-08-18 at 17 00 40@2x" src="https://user-images.githubusercontent.com/1387129/185325709-73ff1bc7-2964-4fcb-8162-baa7cdfba27e.png">

**Environment**
Apple Music:
`Ember | 3.26.2`
Local Reproduction:
`Ember | 3.28.8`
`Ember Data | 3.28.7`

**Additional context**
Loading `4.6.1` using the bookmarklet seems to work
```
javascript: (function() { var s = document.createElement('script'); s.src = '//ember-extension.s3.amazonaws.com/dist_bookmarklet/load_inspector.js'; document.body.appendChild(s); }());
```

Thanks for all the hard work!

**Comment by jlami** [comment]
I have the same problem in Chrome version 104.0.5112.102

Thank you for the bookmarklet workaround btw!

**Comment by steveclifton** [comment]
Same here - came to report both FF and Chrome not working too 🙏

**Comment by andrewobrien** [comment]
Thanks for the quick release, I've just been trying to get it running on `4.6.2` and believe the problem still persists in Firefox

**Comment by andrewobrien** [comment]
Confirming this is fixed in `4.6.3` https://github.com/emberjs/ember-inspector/pull/2108 thanks!
